### DISCUSSION TITLE
February 2021: nooSMTPD libtls-conversion, ciphers, curves and protocols · poolp.org

### DISCUSSION THREAD

**Discussion by poolpOrg** [poolpOrg/poolp.org#127] [Articles]

**Comment by ArchangeGabriel** [comment]
Awesome indeed! Now it makes me wonder whether I should package nooSMPTD in Arch. ;p Quick answer: not now, no time for this (should first fix https://archlinux.org/packages/?sort=&q=&maintainer=Archange&flagged=Flagged —OK, that was much bigger than expected actually… :p), but I might give it a shot in July. ;)
